db/migrate/20210406003421_create_notes.rb

```diff
@@ -0,0 +1,11 @@
+class CreateNotes < ActiveRecord::Migration
+  def change
+    create_table :notes do |t|
+      t.string :title, null: false, default: ""
+      t.text :content, null: false
+      t.references :notable, polymorphic: true
```

Comment:
can this also be not null? or what does it mean if you have a note without a notable?

```suggestion
      t.references :notable, polymorphic: true, null: false
```

Reply:
Oops! Thanks Grant, missed that one. Should have been set as `null: false` in the last commit.